src/components/Book.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+class Book extends React.Component {
```

Comment:
Likewise, `Book` doesn't need to keep local state nor use any component lifecycle methods, we should just keep it as a functional component